homeassistant/components/emulated_hue/hue_api.py

```diff
@@ -517,13 +517,28 @@ def get_entity_state(config, entity):
 
 def entity_to_json(config, entity, state):
     """Convert an entity to its Hue bridge JSON representation."""
+    def format_int(state, field):
```

Comment:
this makes no sense. Our code should just not put None values in `state`

Reply:
We were getting `None` from the get call above. I was thinking this might be a reasonable way to prevent issues like this happening again. If you'd prefer I revert this part of the change, I can do that

Reply:
Please revert. I rather have it raising so we know we have bugs to fix, instead of silently "fixing" things, and not being able to find what is causing behavior down the line.

Reply:
Feedback addressed.